Octokit.Tests.Integration/Clients/IssuesClientTests.cs

```diff
@@ -230,6 +254,14 @@ public IssuesClientTests()
         Assert.True(retrieved.Any(i => i.Number == issue2.Number));
         Assert.True(retrieved.Any(i => i.Number == issue3.Number));
         Assert.True(retrieved.Any(i => i.Number == issue4.Number));
+        var issue = retrieved[0];
+        Assert.Equal(4, issue.Reactions.TotalCount);
+        Assert.Equal(0, issue.Reactions.Plus1);
+        Assert.Equal(0, issue.Reactions.Hooray);
+        Assert.Equal(0, issue.Reactions.Heart);
+        Assert.Equal(0, issue.Reactions.Laugh);
```

Comment:
Similarly, if we're reacting to the most recently updated issue we'd need to look for the `Laugh` reaction here.